pkg/cli/debug/debug.go

```diff
@@ -375,11 +383,67 @@ func (o *DebugOptions) RunDebug() error {
 	if err != nil {
 		klog.V(4).Infof("Unable to get exact template, but continuing with fallback: %v", err)
 	}
+	var debugNodeNS *corev1.Namespace
+	var clusterRoleBinding *rbacv1.ClusterRoleBinding
+	// create namespace only if debugging a node
+	if o.IsNode {
+		debugNodeNS, err = o.CoreClient.Namespaces().Create(context.TODO(), &corev1.Namespace{
```

Comment:
only do this if the ns isn't explicitly set

Reply:
right, fixed that, thanks again